dashboard/db/migrate/20210609011858_set_published_state_for_unit_group_and_script.rb

```diff
@@ -0,0 +1,39 @@
+class SetPublishedStateForUnitGroupAndScript < ActiveRecord::Migration[5.2]
+  def up
+    Script.all.each do |script|
+      script.update!({skip_name_format_validation: true, published_state: script.get_published_state})
+
+      script.write_script_dsl
```

Comment:
@davidsbailey Is there any concern with this running in every environment? Should I make it so the write to script dsl and script json only happen in levelbuilder?

Reply:
There is also a write to the .courses file that happens automatically after the update. I don't know if there is concern there.

Reply:
Good catch! yes, these writes should only be done in levelbuilder_mode.

Reply:
Updated